docs/backups.md

```diff
@@ -0,0 +1,10 @@
+---
+sort: -1 # hide page from menu
+aliases:
+- /operator/backups.html
+---
```

Comment:
These pages and similar ones were previously on VMDocs and consistently appeared on the docs site, but they had become out of sync with broken links. They have now been re-added to the operator so they can be searched and updated.